tests.mk

```diff
@@ -61,17 +61,17 @@ vagrant_test:
 ### go tests
 test:
 	@echo "--> Running go test"
-	@go test -p 1 $(PACKAGES) -tags deadlock
+	@go test -p 1 $(PACKAGES) -tags 'deadlock memdb goleveldb'
 .PHONY: test
 
 test_race:
 	@echo "--> Running go test --race"
-	@go test -p 1 -v -race $(PACKAGES)
+	@go test -p 1 -v -race $(PACKAGES) -tags 'memdb goleveldb'
 .PHONY: test_race
 
 test_deadlock:
 	@echo "--> Running go test --deadlock"
-	@go test -p 1 -v  $(PACKAGES) -tags deadlock
+	@go test -p 1 -v  $(PACKAGES) -tags 'deadlock memdb goleveldb'
```

Comment:
Why should we use `memdb/goleveldb` as `default`? You can use the below:
```
BUILD_TAGS="deadlock memdb goleveldb" make test_tags
```

Reply:
I'll remove it.